### ISSUE TITLE
Inactive blog deletion

### ISSUE THREAD

**Issue by GopalanHarish** [kilimchoi/engineering-blogs#1095]
Wouldn't it be better to move inactive blogs to a new one called "Inactive blogs" rather than deleting its entry from README?
Whats your thought?

**Comment by jacklagare** [comment]
Or label it as `INACTIVE` so that it's still easy to locate it in `README`. Even if it's not being updated anymore, as long as it's still accessible, someone might still find it useful.

**Comment by GopalanHarish** [comment]
exactly, whatever the label is, it's really useful to not remove it completely and the community can also even suggest repo author/create PR to push it to the active list whenever there are new posts.
